launch/pi.launch

```diff
@@ -6,6 +6,12 @@
     <!-- params for hardware interfaces including IMU, GPS, and moteus -->
     <rosparam command="load" file="$(find mrover)/config/esw.yaml"/>
 
+    <rosparam command="load" file="$(find mrover)/config/teleop.yaml"/>
+    <node name="jetson_teleop" pkg="mrover" type="jetson_teleop.py"/>
```

Comment:
Wait are we permanently moving jetson teleop to the Pi? Honestly I don't think it's a terrible idea, then all basic drive/arm functionality would be contained on the pi (I think, correct me if anything is still on the jetson). But if this is the case, we gotta remove this from jetson.launch. Alternatively, maybe you just had this for testing

Reply:
I still have several changes on this branch  that were just for testing since i had to keep switching between PIs. However, we could maybe consider just running it on the pi.